app/services/affiliate-slots-units.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "campaign": "ddb",
+    "category": "ddb-test-unit-id",
+    "tagline": "Suggested For You",
+    "image": "https://static.wikia.nocookie.net/2d18111c-bc45-458f-9898-d2d4e3a50589",
+    "heading": "CLICK ME FOR DDB",
+    "subheading": "DDB subheading",
+    "link": "https://www.dndbeyond.com/",
```

Comment:
Should we add a dummy links field here?

Reply:
Why do you think we should change this?

Reply:
To match our existing affiliate units. Or, is there only going to be one link?

Reply:
Oh I see.